sources/packages/web/src/views/institution/locations/programs/LocationProgramAddEdit.vue

```diff
@@ -93,7 +94,13 @@ export default defineComponent({
             props.programId,
           );
       } else {
-        initNewFormData();
+        // Initialize programData with institution profile data
```

Comment:
Please add a period to the end of the sentence.

Reply:
The period is added. Please have a look.